vllm/sequence.py

```diff
@@ -18,7 +18,7 @@
     from vllm.spec_decode.metrics import SpecDecodeWorkerMetrics
 
 
-@dataclass
+@dataclasses.dataclass
```

Comment:
why is this change needed? can we reduce the number of lines changed to make the review easier?